### ISSUE TITLE
Improve readthedocs formatting

### ISSUE THREAD

**Issue by jakebeal** [SynBioDex/pySBOL3#325]
Something seems to be going wrong in our readthedocs API generation. We'd like it to look more like the python API documentation, and it's just not as pretty as we'd like it to be yet.

**Comment by ahmedtarek26** [comment]
I read about readthedocs then looked at pySBOL3 and python docs
what I found is that the python API documentation has titles and subtitles and each subtitle has an attached link containing the details 
could you tell me tips about the updates I can do with API reference at pySBOL3 doc

**Comment by tcmitchell** [comment]
The goal for this issue is to have the [pySBOL3 documentation](https://pysbol3.readthedocs.io/en/stable/) look more like the Python documentation (https://docs.python.org/3/library/random.html). Specifically, the pySBOL3 API pages under https://pysbol3.readthedocs.io/en/stable/autoapi/index.html, like https://pysbol3.readthedocs.io/en/stable/autoapi/sbol3/attachment/index.html

Is that enough to get started?

**Comment by ahmedtarek26** [comment]
Yeah that's good, I'll start on the issue and tell you the updates
Thanks @tcmitchell

**Comment by sriramsowmithri9807** [comment]
@jakebeal can you mention code!!

**Comment by jakebeal** [comment]
@sriramsowmithri9807 I'm afraid that I'm not quite clear on what you are asking?

**Comment by sriramsowmithri9807** [comment]
@jakebeal, could you clarify the exact error you’re encountering in the API section? 

Here's the [[documentation for the API Reference](https://pysbol3.readthedocs.io/en/stable/autoapi/index.html)](https://pysbol3.readthedocs.io/en/stable/autoapi/index.html) for your reference.
